tests/envlint.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Needs a copyright added.

Also, add a comment describing the purpose of this script.